charts/datasaur/values.yaml

```diff
@@ -160,16 +162,159 @@ jobs:
 
 loki:
   enabled: true
+  alerting:
+    contactPoints.yaml: |
+      apiVersion: 1
+
+      contactPoints:
+        - orgId: 1
+          name: cp_1
+          receivers:
+            - uid: webhook_receiver
+              type: webhook
+              settings:
+                url: https://webhook.site/8bb1d771-b0b3-4867-b680-b67471220f43
+                httpMethod: POST
+            - uid: slack_receiver
+              type: slack
+              settings:
+                url: https://hooks.slack.com/services/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
```

Comment:
is this url secret? if yes, i think this is not okay if we commit it to the public repo

or should we commit it to the values.sample.yaml in the vulcan repo?

Reply:
iya setelah kupikir pikir ini aku jadiin null semua aja, nanti dimasukin ke values.sample.yaml aja buat reference nya user @tonoman3g

Reply:
oke, better gitu